js/recipe_manager.js

```diff
@@ -1,84 +1,119 @@
+const {
+  validateRecipeKey,
+  validateRecipe,
+  isValidKey
+} = require('../js/validation')
+
+const uuidv1 = require('uuid/v1')
```

Comment:
Third-party library requires should go before your namespaces

Reply:
fixed